### PR TITLE
@xtina-starr => Infinite scroll/Display bugs

### PR DESCRIPTION
**Pull request by eessex**
Removes `/news` rendering of `DisplayCanvas` and `RelatedArticlesCanvas` in favor of rendering in Reaction.  Follows up on https://github.com/artsy/reaction/pull/1008.

### CODE REVIEW

File: src/desktop/apps/article/components/__tests__/InfiniteScrollNewsArticle.test.js [hunk 1]
```diff
@@ -160,7 +155,6 @@ describe('<InfiniteScrollNewsArticle />', () => {
     const rendered = shallow(<InfiniteScrollNewsArticle {...props} />)
     await rendered.instance().fetchNextArticles()
     rendered.update()
-    rendered.find(RelatedArticlesCanvas).length.should.equal(1)
     rendered.html().should.containEql('More from Artsy Editorial')
```
Comment (eessex) [inline src/desktop/apps/article/components/__tests__/InfiniteScrollNewsArticle.test.js]: Presence of these components is now tested in reaction, but check for text confirms their presence.

**Comment by eessex** [comment]
Reaction is now bumped and this is ready to go.

File: src/desktop/apps/article/components/NewsArticle.tsx [hunk 2]
```diff
@@ -4,13 +4,16 @@ import Waypoint from 'react-waypoint'
...
+  display?: any
   isActive: boolean
   isFirstArticle: boolean
   isMobile: boolean
   isTruncated: boolean
   nextArticle: any
   onActiveArticleChange: (id: string) => void
   onDateChange: (date: string) => void
+  relatedArticlesForCanvas?: any
+  renderTime?: any
```
Comment (l2succes) [inline src/desktop/apps/article/components/NewsArticle.tsx]: Let's use actual types in our interfaces whenever possible, not obvious if `renderTime` is a `number` or `boolean`.

**Comment by xtina-starr** [comment]
👍

**Pull request merged**